### ISSUE TITLE
rhash generate the wrong magnet for btih

### ISSUE THREAD

**Issue by 0x010** [rhash/RHash#96]
Hello, i found the rhash not generate the right magnet for btih protocol, I am using transmission to valid the generate magnet and torrent.

```
huzhifeng@min ~/tmp $ rhash --magnet --btih --torrent mini.iso
mini.iso.torrent saved
magnet:?xl=76546048&dn=mini.iso&xt=urn:btih:n2ixmnprkf36mrhywaxy36m3um5ymve6
huzhifeng@min ~/tmp $ transmission-show -m mini.iso.torrent
magnet:?xt=urn:btih:6e917635f15177e644f8b02f8df99ba33b86549e&dn=mini.iso

```

see it . the generate torrent , using transission to generate magnet , not match the same of rhash generate

**Comment by ivan386** [comment]
Both magnets are valid and points to same torrent file. Rhash encode info_hash in Base32. Transmission use HEX.

[Magnet URI format](http://bittorrent.org/beps/bep_0009.html#magnet-uri-format)
> \<info-hash>
    Is the info-hash hex encoded, for a total of 40 characters. For compatability with existing links in the wild, clients should also support the 32 character base32 encoded info-hash.

**Comment by 0x010** [comment]
any options here let us generate the same hex ?

?? Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by ivan386** [comment]
Handmade magnet:
```
rhash -p"magnet:?xt=urn:btih:%{btih}&dn=%u"
```

**Comment by rhash** [comment]
Fixed by a922ad250933ae54fff076b0c1aa1702d07686a0

**Comment by netroby** [comment]
@rhash  god like .thanks

**Comment by rhash** [comment]
Fix included in v1.3.9.
